WebRoot/public/extuserprov.jsp

```diff
@@ -67,7 +67,11 @@
 				<table>
 					<tr>
 						<td><app:img id="ZLoginErrorIcon" altkey='ALT_ERROR' src="dwt/ImgCritical_32.png" /></td>
-						<td id="errorMessage"></td>
+						<td id="errorMessage">
+							<% if (request.getAttribute("errorCode") != null) { %>
+								<fmt:message bundle="${zhmsg}" key='${errorCode}'/>
```

Comment:
I have handled message for new errorCode.
https://github.com/Zimbra/zm-web-client/pull/89/files#diff-f88cbfd6558faa3922ca722129e31486R1857

@silentsakky @rohan-ambasta 

Thanks